Makefile

```diff
@@ -10,7 +10,9 @@ TAG = lorax-$(VERSION)-$(RELEASE)
 IMAGE_RELEASE = rhel8-latest
 
 ifeq ($(TEST_OS),)
-TEST_OS = rhel-8-1
+OS_ID = $(shell awk -F= '/^ID=/ {print $$2}' /etc/os-release)
+OS_VERSION = $(shell awk -F= '/^VERSION_ID/ {print $$2}' /etc/os-release)
+TEST_OS = $(OS_ID)-$(OS_VERSION)
```

Comment:
Note that a much simpler approach  is to use something like `$(shell . /etc/os-release; echo "$$ID-$$VERSION_ID")`; but I don't want to quibble over such implementation details, particularly because this is a backport (i. e. if you want to change it, change it in master first).

However, this breaks on our RHEL images. For hysterical raisins they are *not* called "rhel-8.2", but "rhel-8-2". I. e. you need to add an additional `tr '.' '-'` (or some equivalent `sed`).

Reply:
fixed